tests/test_frame.py

```diff
@@ -610,7 +610,10 @@ def test_types_idxmax() -> None:
 # This was added in 1.1.0 https://pandas.pydata.org/docs/whatsnew/v1.1.0.html
 def test_types_value_counts() -> None:
     df = pd.DataFrame(data={"col1": [1, 2], "col2": [1, 4]})
-    df.value_counts()
+    c1: Series[int] = df.value_counts()
+    check(c1, pd.Series, np.integer)
+    c2: Series[float] = df.value_counts(normalize=True)
+    check(c2, pd.Series, float)
```

Comment:
Can you use the `check(assert_type(...))` pattern here?  We're requiring that on any new tests.